apps/drec-ui/src/organization/containers/file/DownloadOrgDocs/DownloadOrgDocs.tsx

```diff
@@ -0,0 +1,28 @@
+import { DownloadableChip } from '@energyweb/origin-ui-core';
+import React, { FC } from 'react';
+import { fileDownloadHandler } from '../../../../api';
+import { useStyles } from './DownloadOrgDocs.styles';
+
+export interface DownloadOrgDocsProps {
+    documents: string[];
+    blockTitle: string;
+}
+
+export const DownloadOrgDocs: FC<DownloadOrgDocsProps> = ({ documents, blockTitle }) => {
+    const classes = useStyles();
+    return (
+        <aside className={classes.thumbsContainer}>
+            {documents?.map((documentId, index) => (
```

Comment:
Props say that it can be array only, so nullish is not required

Reply:
Not sure if we are still sending an empty array in case docs were not uploaded on the org registration. 
But if yes - you are right

Reply:
If we are not sending empty array in such case, then type has to be corrected ;)